docs/source/dev-get-started.md

```diff
@@ -0,0 +1,129 @@
+# Getting Started
```

Comment:
For some reason those are not showing up. Is it because it's not deployed yet?
<img width="1001" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/999878cf-896b-44a0-be0c-426154399000">